src/langcheck/metrics/prompts/en/metrics/answer_correctness.j2

```diff
@@ -0,0 +1,18 @@
+You are evaluating the correctness of the answer to a user's query. Here is the data:
```

Comment:
nit: even for humans it could be a bit confusing if the prompt has both "answer" and "reference answer"... Could we mention what this "reference answer" is here? (For Japanese too)

Reply:
Yep fair point. I rewrote the prompt - PTAL!